From 0c72e11fee13749f697ab87d9f6b04c835696657 Mon Sep 17 00:00:00 2001
From: Allan Sandfeld Jensen <allan.jensen@qt.io>
Date: Mon, 10 Dec 2018 14:32:13 +0100
Subject: [PATCH] Fix x86-32 builds with GCC 8

The C++11 alignof() now returns minimum alignment and not
prefered alignment which Chromium expects.

Change-Id: I7aadb426e6e15b4f2317cafdde550ea74e4c1ccf
Fixes: QTBUG-72391
Reviewed-by: Michal Klocek <michal.klocek@qt.io>
---
 src/3rdparty/chromium/mojo/public/c/system/macros.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/3rdparty/chromium/mojo/public/c/system/macros.h b/src/3rdparty/chromium/mojo/public/c/system/macros.h
index 6de4a913de..87016c431d 100644
--- a/src/3rdparty/chromium/mojo/public/c/system/macros.h
+++ b/src/3rdparty/chromium/mojo/public/c/system/macros.h
@@ -27,10 +27,10 @@
   (sizeof(void*) == 4 ? 32 : 0)
 
 // Like the C++11 |alignof| operator.
-#if __cplusplus >= 201103L
-#define MOJO_ALIGNOF(type) alignof(type)
-#elif defined(__GNUC__)
+#if defined(__GNUC__)
 #define MOJO_ALIGNOF(type) __alignof__(type)
+#elif __cplusplus >= 201103L
+#define MOJO_ALIGNOF(type) alignof(type)
 #elif defined(_MSC_VER)
 // The use of |sizeof| is to work around a bug in MSVC 2010 (see
 // http://goo.gl/isH0C; supposedly fixed since then).
-- 
2.16.3

